pkg/runtime/wfengine/sqlite_backend.go

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
Can you move this file to `pkg/wfengine/sqlite/backend.go`? It's nice to have a folder for each component. We can then keep the metadata.go file in the same folder

Reply:
I wanted to do the same, but I had to give sqlite as package and then it creates cyclic dependency wfengine, so I had to keep it this way. It doesn't allow to give package as wfengine inside sqlite folder